bon/tests/integration/builder/target_feature.rs

```diff
@@ -0,0 +1,42 @@
+#[cfg(target_arch = "x86_64")]
+#[rustversion::since(1.86.0)]
+mod msrv_1_86 {
+    #![allow(dead_code)]
+    use crate::prelude::*;
+
+    #[test]
+    fn target_feature_fn() {
```

Comment:
`[nit]`

By convention, the test functions are called `test_struct`, `test_function` and `test_method` for different variants of syntax.